python/cuml/test/test_tsne.py

```diff
@@ -107,3 +107,12 @@ def test_tsne_large(nrows, ncols):
     Y = tsne.fit_transform(X)
     nans = np.sum(np.isnan(Y))
     assert nans == 0
+
+
+def test_components_exception():
+    X, y = make_blobs()
+
+    X = X.astype(np.float32)
```

Comment:
we do no need to generate the dataset since building the estimator will raise the error, right?

Reply:
Good point! I'll get rid of that